src/main/java/com/savvato/tribeapp/services/ToBeReviewedChecker.java

```diff
@@ -0,0 +1,45 @@
+package com.savvato.tribeapp.services;
+
+import com.savvato.tribeapp.entities.RejectedPhrase;
+import com.savvato.tribeapp.entities.ToBeReviewed;
+import com.savvato.tribeapp.repositories.RejectedPhraseRepository;
+import com.savvato.tribeapp.repositories.ReviewSubmittingUserRepository;
+import com.savvato.tribeapp.repositories.ToBeReviewedRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.logging.Logger;
+
+@Service
+public class ToBeReviewedChecker {
+    @Autowired
+    ToBeReviewedRepository toBeReviewedRepository;
+    @Autowired
+    RejectedPhraseRepository rejectedPhraseRepository;
+
+    @Autowired
+    ReviewSubmittingUserRepository reviewSubmittingUserRepository;
+    static final Logger LOGGER = Logger.getLogger(ToBeReviewedChecker.class.getName());
+
+    @Scheduled(fixedDelayString = "PT10M")
+    public void updateUngroomedPhrases() {
+        LOGGER.info("Beginning updateUngroomedPhrases process...");
+        List<ToBeReviewed> ungroomedPhrases = toBeReviewedRepository.getAllUngroomed();
+
+        for (ToBeReviewed tbr : ungroomedPhrases) {
+            Optional<RejectedPhrase> matchingRejectedPhrase = rejectedPhraseRepository.findById(tbr.getId());
+            if (matchingRejectedPhrase.isEmpty()) {
+                // contact wordsApi...
+            }
+            else {
+                LOGGER.warning("Phrase has already been rejected!");
```

Comment:
Try catch an exception here, and log the call stack as an Error... We shouldn't have an item in TBR if it's phrase appears in rejectedPhrase table, we should have checked that earlier, so if it happens here let's be loud about it so we know something is amiss.

Reply:
..and I just re-read the requirements on TRIB-77, and I see that you did exactly what I asked in the ticket. Thank you for that. My comment above only became clear to me after I read your code.